freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/model/ServiceKeytabRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.freeipa.api.v1.kerberosmgmt.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.freeipa.api.v1.kerberosmgmt.doc.KeytabModelDescription;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("ServiceKeytabV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ServiceKeytabRequest {
+
+    @ApiModelProperty(KeytabModelDescription.ID)
+    private Long id;
```

Comment:
for this ID is the same too

Reply:
I have added to track the request/response. Will remove it.